source/blog/2017-08-16-ovirt-ansible-roles-how-to-use.html.md

```diff
@@ -0,0 +1,153 @@
+---
+title: How to use oVirt Ansible roles
+author: omachace
+tags: Ansible, Roles, Automation, Configuration management
+date: 2017-08-16 15:00:00 CET
+comments: true
+published: true
+---
+
+The recent post, [An Introduction to Ansible Roles](./2017-07-19-ovirt-ansible-roles-an-introduction.html.md), covered the new roles introduced in the oVirt 4.1.6 release. This follow-up post will explain how to set up and use Ansible roles either using Ansible Galaxy or oVirt Ansible Roles RPM.
+
+## Ansible Galaxy
+
+[Ansible Galaxy](https://galaxy.ansible.com/) stores many different Ansible roles, making your life simpler. oVirt Ansible roles are also stored in Galaxy. To install those roles from Ansible Galaxy, perform the following steps:
+
+To install roles on your local machine, run the following command:
+
+```bash
+$ ansible-galaxy install ovirt.ovirt-ansible-roles
+```
+
+This will install your roles into directory `/etc/ansible/roles/ovirt.ovirt-ansible-roles/`. By default, Ansible only searches for roles in `/etc/ansible/roles/` directory and your current working directory. To change the directories where Ansible looks for roles, modify the `roles_path` option of `[defaults]` section in `ansible.cfg` configuration file. The default location of this file is in `/etc/ansible/ansible.cfg`.
```

Comment:
Please change to (one paragraph. Separated here to highlight the changes).
"This will install your roles into the directory: `/etc/ansible/roles/ovirt.ovirt-ansible-roles/`."

"By default, Ansible only searches for roles in the `/etc/ansible/roles/` directory and your current working directory. 

"To change where Ansible looks for roles, modify the `roles_path` option of the `[defaults]` section in the `ansible.cfg` configuration file."

"The default location of this file is: `/etc/ansible/ansible.cfg`."